coalib/parsing/DefaultArgParser.py

```diff
@@ -72,6 +72,12 @@ def default_arg_parser(formatter_class=None):
                             help='Files that will be analyzed will be '
                                  'restricted to those in the globs listed '
                                  'in this argument as well the files setting')
+    arg_parser.add_argument('--caching',
```

Comment:
Huh, never thought about it. I personally think something like `--flush-cache` would work better in conveying its meaning.